.github/workflows/ci-linkchecks.yml

```diff
@@ -13,20 +13,13 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: "Generate token"
-        uses: actions/create-github-app-token@v2
-        id: generate-token
-        with:
-          app-id: ${{ secrets.AUTH_APP_ID }}
-          private-key: ${{ secrets.AUTH_APP_PRIVATE_KEY }}
-
       - name: Link Checker
         id: lychee
         uses: lycheeverse/lychee-action@1d97d84f0bc547f7b25f4c2170d87d810dc2fb2c
         with:
-          token: ${{ steps.generate-token.outputs.token }}
+          token: ${{secrets.GITHUB_TOKEN}}
           fail: false
-          args: "--verbose './docs/**/*.rst' './docs/**/*.inc' './lib/**/*.py'"
+          args: "--verbose --max-concurrency 1 './docs/**/*.rst' './docs/**/*.inc' './lib/**/*.py'"
```

Comment:
We're in no hurry since this action is running overnight.

Preventing concurrency (which defaulted to 128) should help reduce the chance of rate limiting, either from GitHub or other providers.